src/test/java/edu/oakland/test/database/DatabaseGisInterfaceTest.java

```diff
@@ -0,0 +1,61 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.production.database.DatabaseGisInterfaceImplementation;
+import edu.oakland.production.database.DatabaseGisManagerStub;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import java.util.Random;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database GisInterface Test")
+public class DatabaseGisInterfaceTest {
+
+  @Test
+  @DisplayName("LDP going in the same LDP coming out")
+  void dataPointInIsDataPointOut() {
+    DatabaseGisManagerStub dgms = new DatabaseGisManagerStub();
+    DatabaseGisInterfaceImplementation dgi = new DatabaseGisInterfaceImplementation(dgms);
+    LocationDataPoint ldp = new LocationDataPoint(0, 0, LocalDateTime.of(
+        (int) (Math.random() * 50 + 1970),
+        (int) (Math.random() * 12 + 1),
+        (int) (Math.random() * 28 + 1),
+        (int) (Math.random() * 24),
+        (int) (Math.random() * 60)
+    ));
+    int i = 0;
+    dgi.receiveStoreRequest(ldp);
+    assertEquals(null, dgms.getLocationDataPoint(i));
```

Comment:
This test should be failing! You're test is saying that `LocationDataPoint` coming out is always null, which is not true!